libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -354,6 +354,10 @@ void AP_SerialManager::init()
                     state[i].uart->set_unbuffered_writes(true);
                     state[i].uart->set_flow_control(AP_HAL::UARTDriver::FLOW_CONTROL_DISABLE);
                     break;
+
+                default:
```

Comment:
why default instead of the specific protocol?

Reply:
I wrote that first, but the code under it isn't actually doing anything specific to the protocol so I changed to default. I can change it back although it looks like a good idea to initialize the UART port with the parameters the user has set.